### ISSUE TITLE
Display inventory numbers in PDFs

### ISSUE THREAD

**Issue by dertel** [dertel/old-issues#316]
**[Original report](https://bitbucket.org/davidartrecord/artrecord/issues/316) by Madeleine Paré (Bitbucket: [madeleineArtRec](https://bitbucket.org/madeleineArtRec), ).**

----------------------------------------

**Comment by dertel** [comment]
**Original comment by David Ertel (Bitbucket: [davidartrecord](https://bitbucket.org/davidartrecord), GitHub: [dertel](https://github.com/dertel)).**

----------------------------------------

So you want the inv number to always appear in the set edit page for each work caption?

And then we make it a share option and a pdf option?

**Comment by dertel** [comment]
**Original comment by David Ertel (Bitbucket: [davidartrecord](https://bitbucket.org/davidartrecord), GitHub: [dertel](https://github.com/dertel)).**

----------------------------------------

related to #310

**Comment by dertel** [comment]
**Original comment by David Ertel (Bitbucket: [davidartrecord](https://bitbucket.org/davidartrecord), GitHub: [dertel](https://github.com/dertel)).**

----------------------------------------

@madeleineArtRec  pinging you on this question <3

**Comment by dertel** [comment]
**Original comment by Madeleine Paré (Bitbucket: [madeleineArtRec](https://bitbucket.org/madeleineArtRec), ).**

----------------------------------------

Yes to the first comment! This is not a rush though.

**Comment by dertel** [comment]
**Original comment by David Ertel (Bitbucket: [davidartrecord](https://bitbucket.org/davidartrecord), GitHub: [dertel](https://github.com/dertel)).**

----------------------------------------

I've implemented the option to display inventory numbers in PDFs. I've pushed it to dev and will launch it with the advanced search. 

Since this feature is low priority, I will wait to implement the display of  inventory # on Set pages—the sharing options are kind of complicated and not quick to implement.
